snugget_load.py

```diff
@@ -147,12 +140,23 @@ def getFilterFieldName(shapefile):
 
 def getShapefileFilter(shapefile, filterVal):
   fieldName = getFilterFieldName(shapefile)
-  kwargs = {fieldName: filterVal}
-  if shapefile.objects.filter(**kwargs).exists():
-    return shapefile.objects.get(**kwargs)
+  if fieldName == 'rast':
+    for tile in shapefile.objects.defer("rast").all():
```

Comment:
Note that this is a very time-consuming function which currently gets called more than once per row that references a raster file.  I think it's going to be worth refactoring to call this once per row and pass the result around.

Reply:
Fixed by my latest commit.